vertical-pod-autoscaler/pkg/admission-controller/resource/vpa/handler_test.go

```diff
@@ -78,6 +87,40 @@ func TestValidateVPA(t *testing.T) {
 			},
 			expectError: fmt.Errorf("unexpected UpdateMode value bad"),
 		},
+		{
+			name: inPlaceOrRecreateDisabledTestNames[0],
```

Comment:
I think this is way more readable if you just inline the description here, I wouldn't worry much about extracting this into another variable.

Same below.